upgrades/upgrade_testing_framework/steps/step_select_expectations.py

```diff
@@ -7,11 +11,22 @@ class SelectExpectations(FrameworkStep):
 
     def _run(self):
         # Get the state we need
+        source_version = self.state.test_config.clusters_def.source.engine_version
+        target_version = self.state.test_config.clusters_def.target.engine_version
+        # path_to_knowledge_base = self.state.test_config.path_to_knowledge_base
+        path_to_knowledge_base = Path("../knowledge_base")
 
         # Begin the step body
-        # Query the expectation knowledge base
-        self.logger.info("Querying applicable expectations...")
+        # Load all expectations
+        self.logger.info("Loading Knowledge Base and filtering for relevant expectations")
+        expectations = load_knowledge_base(path_to_knowledge_base)
+        # Filter for relevant ones
+        relevant_expectation_ids = [e.id for e in expectations
+                                    if e.is_relevant_to_version(source_version) or
+                                    e.is_relevant_to_version(target_version)]
+        self.logger.info(f"Found {len(relevant_expectation_ids)} relevant expectations.")
+        self.logger.debug(f"Relevant expectations: {relevant_expectation_ids}")
```

Comment:
Good call; probably will end up overwhelming to put all of them to STDOUT by name.